src/RevProc.cc

```diff
@@ -1262,14 +1259,11 @@ bool RevProc::PrefetchInst(){
 }
 
 RevInst RevProc::DecodeInst(){
```

Comment:
I would suggest we rename this to "FetchAndDecodeInst()" or something similar as we now have a DecodeInst() that only does Decode

Reply:
> I would suggest we rename this to "FetchAndDecodeInst()" or something similar as we now have a DecodeInst() that only does Decode

Done